packages/rpc-core/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
     "$schema": "https://json.schemastore.org/tsconfig",
     "compilerOptions": {
-        "lib": ["DOM", "ES2017", "ES2020.BigInt", "ES2022.Error"]
+        "lib": ["DOM", "ES2017", "ES2020.BigInt", "ES2022.Error"],
+        "resolveJsonModule": true
```

Comment:
@mcintyre94 is adding this to `rpc-core`'s `tsconfig.json` an issue?

Reply:
I assume it's required for importing that JSON keypair? Honestly I don't know but I can't see any harm, or any impact on consumers, looks like it just affects our TS compilation.

Reply:
Me too but I didn't want it to somehow affect bundle size or anything related. I tried adding it to the `tsconfig.json` within the test-configs dir but that didn't seem to solve it

Reply:
I don't see it anywhere in the bundle and from my initial research it appears it only adds JSON modules to the bundle if they're imported in TS code that is also added to the bundle.

So, types and tests really shouldn't affect anything if pulled from JSON

Reply:
That makes sense to me, I wouldn't expect it to make any difference in code that isn't importing JSON